packages/react-native/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@aemforms/af-react-native",
+  "version": "0.0.1",
+  "description": "React Spectrum based componets form headless",
+  "author": "Adobe Systems",
+  "license": "Adobe Proprietary",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib",
+    "LICENSE"
+  ],
+  "publishConfig": {
+    "registry": "https://artifactory.corp.adobe.com/artifactory/api/npm/npm-aem-release-local",
+    "access": "public"
+  },
+  "scripts": {
+    "clean": "rm -rf lib target",
+    "eslint": "npx eslint src/**/*.ts*",
+    "build": "npm run clean && npm run eslint && npx tsc",
+    "test:ci": "jest --silent --coverage",
+    "prepublishOnly": "npm run build && npm run test",
+    "test": "jest --silent"
+  },
+  "dependencies": {
+    "@aemforms/af-core": "^0.22.19",
```

Comment:
Please use latest version of af-core here

Reply:
Done